Makefile

```diff
@@ -181,7 +181,7 @@ publish_kademlia_deb:
 
 publish_deb:
 	@if [ $(AWS_ACCESS_KEY_ID) ] ; then \
-		if [ "$(CIRCLE_BRANCH)" = "master" ] && [ "$(CIRCLE_JOB)" = "build-artifacts--testnet_postake" ] ; then \
+		if [ "$(CIRCLE_BRANCH)" = "master" ] && [ "$(CIRCLE_JOB)" = "build-artifacts--testnet_postake" || "$(CIRCLE_JOB)" = "build-artifacts--testnet_postake_many_proposers" ] ; then \
```

Comment:
aren't the two jobs going to fight over what the latest `coda-*` package are?

Reply:
no, they are named by profile, so there's no conflict.